docs/llb/llb-api-spec.md

```diff
@@ -662,6 +662,8 @@ The announcements were retrieved successfully.
       "id": ID,
       "title": STRING,
       "description": STRING,
+      "eventId": ID OR NULL,
```

Comment:
Same comment here - are you sure `eventId` is included?

Reply:
Yeah I guess it does. See [comment](https://github.com/Code-4-Community/c4c-dev-docs/pull/48/files/68cd8276192699d76b853e19c9e7ff1d0fc7b3a2#r586929153)